### ISSUE TITLE
how to add a parameter that can be trained in custom loss function

### ISSUE THREAD

**Issue by yz-ju** [open-mmlab/mmdetection#10560]
Now, I want to try a new loss function,but there is a parameter that need be trained.
How to add a parameter that can be trained in custom loss function?

**Comment by hhaAndroid** [comment]
@yz-ju You need to initialize it with the `nn.Parameter` class in the model

**Comment by MR-hyj** [comment]
Try:
```python
class MyLoss(nn.Module):
    def __init__(self, **args):
        self.para = nn.Parameter(...)
    
    def forward(self, pred, gt):
        pass
```
